modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -2,137 +2,312 @@
+--
++
 [NOTE]
 ====
-You can run the following commands to get the full host name of the {ocp-short} image registry, and then use the host name in a command to log in:
-
-[source,terminal]
+Running the `oc-mirror` command creates a local workspace containing the Helm chart and a `ImageContentSourcePolicy` (ICSP) manifest. The ICSP manifest contains an automatically-generated `imageContentSourcePolicy.yaml` file that you must apply against the cluster in a later step.
+====
++
+.Example output
+[source,terminal,subs="attributes+"]
 ----
-REGISTRY_HOST=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
+Writing image mapping to oc-mirror-workspace/results-1738070846/mapping.txt
+Writing ICSP manifests to oc-mirror-workspace/results-1738070846
 ----
-
-[source,terminal]
++
+. In your workspace, locate the `imageContentSourcePolicy.yaml` file by running the `ls` command. For example:
++
+[source,terminal,subs="attributes+"]
 ----
-podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
+ls <workspace-directory>/<results-directory>
 ----
-====
++
+--
+where:
 
-. Log in to the `registry.redhat.io` in `podman` by running the following command:
+`<workspace-directory>` :: Specifies the name of your workspace directory, for example, `oc-mirror-workspace`.
+
+`<results-directory>` :: Specifies the name of your results directory, for example, `results-1738070846`.
+--
 +
-[source,terminal]
+. To mirror the Helm chart, deploy the `imageContentSourcePolicy.yaml` file in the disconnected cluster by running the `oc apply` command. For example:
++
+[source,terminal,subs="attributes+"]
 ----
-podman login registry.redhat.io
+oc apply -f <workspace-directory>/<results-directory>/<image-content-policy-file>
 ----
 +
-For more information about registry authentication, see https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
+--
+where:
+
+`<workspace-directory>` :: Specifies the name of your workspace directory, for example, `oc-mirror-workspace`.
 
-. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation, by running the following commands:
+`<results-directory>` :: Specifies the name of your results directory, for example, `results-1738070846`.
+
+`<results-directory>` :: Specifies the name of the image content source policy file, for example, `ImageContentSourcePolicy.yaml `.
+--
+. In your air-gapped environment, deploy the Helm chart to the namespace that you want to use by running the `helm install` command and `namespace` options. For example:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal,subs="attributes+"]
 ----
-podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version}
+helm install <rhdh-instance> <workspace-directory>/<results-directory>/<charts-directory>/<archive-file> --namespace <your-namespace> --create-namespace
```

Comment:
```suggestion
CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')

helm install <rhdh-instance> <workspace-directory>/<results-directory>/<charts-directory>/<archive-file> --namespace <your-namespace> --create-namespace \
  --set global.clusterRouterBase="$CLUSTER_ROUTER_BASE"
```

I would suggest passing the `clusterRouterBase` parameter right when deploying the RHDH instance from the Helm Chart.
Otherwise, some things like the Home Page won't work if the user tries to access the RHDH instance from a browser.